src/rt-app_args.c

```diff
@@ -43,8 +43,8 @@ char help_full[] = \
 "through the standard input.\n\n"
 "Miscellaneous:\n"
 "  -v, --version      display version information and exit\n"
-"  -l, --log          set verbosity level (10: ERROR/CRITICAL, 50: NOTICE (default)\n"
-"                                    75: INFO, 100: DEBUG)\n"
+"  -l, --log          set verbosity level (10: error/critical, 50: notice (default)\n"
```

Comment:
use either error or critical but error/critical is pointless and can be confusing: we can wonder if we have to write "error", "critical" or "error/critical"

Reply:
yes, i was wondering about this as well but decided to keep the original values. there is some confusion with your proposed solution, it wouldn't be clear which level is what, though.

how about `(10: critical, 20: error, 50: notice (default) ...)` ? then it would be clear what the string should be and what the values are?

Reply:
ping...

Reply:
my goal is to remove the level associated to the string from the help. if you use the string, you don't care about the value but the functional level

Reply:
as in `(critical, error, notice (default), ....)`, just without the numbers? yes, could remove them, if they bother you. could you please confirm?

i assume 'error' should still get a new value, as proposed above - ?

Reply:
yes for both questions.

Reply:
ok